src/cljdoc/server/search/shared.clj

```diff
@@ -0,0 +1,6 @@
+(ns cljdoc.server.search.shared
+  (:import (org.apache.lucene.store FSDirectory)
+           (java.nio.file Paths)))
+
+(defn ^FSDirectory fsdir [index-dir]
+  (FSDirectory/open (Paths/get index-dir (into-array String nil))))
```

Comment:
Unless there's a reason that this is in its own namespace I'd suggest to move it into the other namespace.

Reply:
well, we need to use it in both in `search` and `artifact-indexer` ns. The only alternative is to duplicate it in both.